vesta/tables/clash_of_code.py

```diff
@@ -0,0 +1,28 @@
+from typing import Optional
+
+import sqlalchemy as db
+
+from . import Base
+from .. import session_maker
+from ..services import clash_of_code_helper, ClashOfCodeGame
+
+session = session_maker()
+
+class ClashOfCodeGuildGame(Base):
+    __tablename__ = "clash_of_code_guild_game"
+
+    guild_id = db.Column(db.BigInteger, nullable=False, primary_key=True)
+    last_clash_id = db.Column(db.String(511), nullable=True)
```

Comment:
Is there a need for a so long ID? verify the actual size before setting random values

Reply:
Will fix